src/services/codefixes/fixJSDocTypes.ts

```diff
@@ -8,11 +8,16 @@ namespace ts.codefix {
     function getActionsForJSDocTypes(context: CodeFixContext): CodeAction[] | undefined {
         const sourceFile = context.sourceFile;
         const node = getTokenAtPosition(sourceFile, context.span.start, /*includeJsDocComment*/ false);
-        const decl = ts.findAncestor(node, n => n.kind === SyntaxKind.VariableDeclaration);
+        const decl = ts.findAncestor(node,
+                                     n => n.kind === SyntaxKind.VariableDeclaration ||
+                                     n.kind === SyntaxKind.Parameter ||
+                                     n.kind === SyntaxKind.PropertyDeclaration ||
+                                     n.kind === SyntaxKind.PropertyAssignment);

```

Comment:
When will you have a `PropertyAssignment` with a type annotation?

Reply:
My mistake, I was thinking of PropertySignature